javascript/pigeonholes.js

```diff
@@ -0,0 +1,183 @@
+var rowValues = [2, 3, 4]

```

Comment:
Also I'm not really sure why there are +s at the beginning of each line and no ; at the end? Is it supposed to be like that?